vernac/vernacentries.ml

```diff
@@ -1016,14 +1016,16 @@ let vernac_cofixpoint ~atts ~pm discharge l =
 
 let vernac_scheme l =
   if Dumpglob.dump () then
```

Comment:
This is unrelated to the PR, but incidentally, this test is done in `Dumglob` already, so better to remove it I think.

If you consider that it is too unrelated, it is also ok.

Reply:
I think that is worth taking care of.

Reply:
Are you suggesting I remove all conditionals for Dumpglob.dump () then?

Reply:
It seems so. One could double-check with a test but only a few of commands have a `if Dumpglob.dump ()` already, and the core "dumping" functions `Dumpglob.dump_string` and `Dumpglob.start_dump_glob` have their own test, so it should work. In any case, the current inconsistency is a source of confusion.

Reply:
There are quite a few of these in the whole file. Are you suggesting I remove all of those?

Reply:
Actually, for separation of concerns, might be better to do it in a separate PR then.